bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml

```diff
@@ -318,20 +318,19 @@ spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
-    - description: CommonService is the Schema for the commonservices API
-      displayName: CommonService
-      kind: CommonService
-      name: commonservices.operator.ibm.com
-      version: v3
-      specDescriptors:
-      - description: The size of deployment profile for IBM Cloud Pak foundational services
-        displayName: Size
-        type: string
-        path: size
-        x-descriptors:
-          - 'urn:alm:descriptor:com.tectonic.ui:select:small'
-          - 'urn:alm:descriptor:com.tectonic.ui:select:medium'
-          - 'urn:alm:descriptor:com.tectonic.ui:select:large'
+      - description: CommonService is the Schema for the commonservices API
+        displayName: Common Service
```

Comment:
I believe we need to use `displayName: CommonService`

Reply:
weird, I thought I rebased from master already